bin/tests/test_data/titanic/process_titanic_to_stimulus.py

```diff
@@ -0,0 +1,44 @@
+import polars as pl
+import argparse
+
+def arg_parser():
+    parser = argparse.ArgumentParser(description="Process Titanic dataset to stimulus format")
+    parser.add_argument("--input", type=str, help="Path to input csv file, should be identical to Kaggle download of the Titanic dataset, see : https://www.kaggle.com/c/titanic/data", required=True)
+    parser.add_argument("--output", type=str, help="Path to output csv file", default="titanic_stimulus.csv", required=False)
+    return parser.parse_args()  
+
+def main():
+    args = arg_parser()
+    df = pl.read_csv(args.input)
+    df = df.select([
+        "PassengerId",
+        "Survived",
+        "Pclass",
+        "Sex",
+        "Age",
+        "SibSp",
+        "Parch",
+        "Fare",
+        "Embarked"
+    ])
+
+    df = df.drop_nulls()
+
+    # Rename columns to match stimulus format
+
+    df = df.rename({"Survived": "survived:label:int_class",
```

Comment:
why you have int_class, int_reg, but only float?
Also what is the difference between str_class and str?

Reply:
This is to illustrate how someone could implement different "type names", int_class for instance gets binarized, and int_reg gets transformed into ranks (see TitanicExperiment in experiments.py for example)